### PR TITLE
Set page size #94

### PR DESCRIPTION
**Pull request by vyankatesh24**
# Description

Added new Enhancement option to change Page size

Fixes #(issue)
#94 

## Type of change
Just put an x in the [] which are valid.
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes.
- [x] `./gradlew assembleDebug assembleRelease`
- [x] `./gradlew checkstyle`

### CODE REVIEW

**Comment by Swati4star** [comment]
The A0 to A10 and the spinner by it's side are not aligned. Same for B0 to B10

And have you tried the functionality? The final PDF is being made according to given size?

File: app/src/main/java/swati4star/createpdf/fragment/HomeFragment.java [hunk 1]
```diff
@@ -215,7 +220,7 @@ public void onInput(@NonNull MaterialDialog dialog, CharSequence input) {
                             FileUtils utils = new FileUtils(mActivity);
                             if (!utils.isFileExist(filename + getString(R.string.pdf_ext))) {
                                 new CreatePdf(mActivity, mImagesUri, filename, mPassword, mQuality,
-                                        HomeFragment.this).execute();
+                                        HomeFragment.this, mPageSize).execute();
```
Comment (Swati4star) [inline app/src/main/java/swati4star/createpdf/fragment/HomeFragment.java]: Keep homeFragment.this as last parameter only.

File: app/src/main/java/swati4star/createpdf/fragment/HomeFragment.java [hunk 2]
```diff
@@ -405,11 +413,71 @@ public void onItemClick(int position) {
...
+                .positiveText(android.R.string.ok)
+                .negativeText(android.R.string.cancel)
+                .onPositive(new MaterialDialog.SingleButtonCallback() {
+                    @Override
+                    public void onClick(@NonNull MaterialDialog dialog, @NonNull DialogAction which) {
+                        View view = dialog.getCustomView();
+                        RadioGroup radioGroup = view.findViewById(R.id.radio_group_page_size);
+                        int selectedId = radioGroup.getCheckedRadioButtonId();
+
+                        String s;
```
Comment (Swati4star) [inline app/src/main/java/swati4star/createpdf/fragment/HomeFragment.java]: Change the name s to something appropriate

File: app/src/main/java/swati4star/createpdf/util/CreatePdf.java [hunk 3]
```diff
@@ -43,15 +42,17 @@
     private final ArrayList<String> mImagesUri;
     private final Activity mContext;
     private final OnPDFCreatedInterface mOnPDFCreatedInterface;
+    private Rectangle mPageSize;
 
     public CreatePdf(Activity context, ArrayList<String> imagesUri, String fileName, String password,
-                     String qualityString, OnPDFCreatedInterface onPDFCreated) {
+                     String qualityString, OnPDFCreatedInterface onPDFCreated, Rectangle pageSize) {
```
Comment (Swati4star) [inline app/src/main/java/swati4star/createpdf/util/CreatePdf.java]: Change parameter sequence. Keep interface at end only

**Comment by vyankatesh24** [comment]
@Swati4star 
I have tested functionality it's working.
The dimensions of pdf are changing as per input.

**Comment by vyankatesh24** [comment]
image

**Comment by Swati4star** [comment]
Awesome @vyankatesh24 !!
I have mentioned a few code refactoring & UI changes. Just do those & I'll merge the PR.

**Comment by vyankatesh24** [comment]
Yeah Ok

**Comment by vyankatesh24** [comment]
@Swati4star 
Please have a look 
Do they looks in a line now


image

**Comment by Swati4star** [comment]
Yes @vyankatesh24

**Pull request merged**